.github/runlint.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+###############################################################################
+# Copyright (c) 2021, 2021 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+###############################################################################
+
+# Exit immediately if any unexpected error occurs.
+set -e
+# trace the commands that are run
+set -x
+
+MAKE_JOBS=4
```

Comment:
There is a comment in `build-on-travis.sh` that explains why `4` is chosen.
Do we know the rough specifications of machines that will run this script?

Reply:
From https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners -> https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1604-README.md

Reply:
According to the second link, jdk11 is available: can we use that rather than downloading another copy?

Reply:
Yes, just check. Same as travis-ci. 2 cores. I will add a similar comment for the chosen reason. 

```
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                2
On-line CPU(s) list:   0,1
Thread(s) per core:    1
Core(s) per socket:    2
Socket(s):             1
NUMA node(s):          1
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 85
Model name:            Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
Stepping:              4
CPU MHz:               2095.197
BogoMIPS:              4190.39
Hypervisor vendor:     Microsoft
Virtualization type:   full
L1d cache:             32K
L1i cache:             32K
L2 cache:              1024K
L3 cache:              36608K
NUMA node0 CPU(s):     0,1
```

Reply:
Updated the comment and use exising jdk11 - JAVA_HOME_11_X64